docs/output.md

```diff
@@ -80,6 +88,7 @@ Antimicrobial Resistance Genes (ARGs):
 - [DeepARG](#deeparg) – antimicrobial resistance gene detection, using a deep learning model.
 - [fARGene](#fargene) – antimicrobial resistance gene detection, using Hidden Markov Models.
 - [RGI](#rgi) – antimicrobial resistance gene detection, based on alignment to the CARD database.
+- [argNorm](#argNorm) - Normalize ARG annotations to the ARO
```

Comment:
```suggestion
- [argNorm](#argNorm) – Normalize ARG annotations to the ARO
```
Long dash, sorry for the details :wink: You can also commit reviewer suggestion directly on Github, so you won't miss such details/have typos from doing it manually via CLI.